src/pybind/mgr/dashboard/module.py

```diff
@@ -832,6 +832,13 @@ def get_perf_schema(self, **args):
             'engine.autoreload.on': False
         })
 
+        # Publish the URI that others may use to access the service we're
```

Comment:
Use "we are" instead of we're as per coding guidelines. See
https://github.com/ceph/ceph/pull/16705/files